### DISCUSSION TITLE
Computing Wake Superposition in FLORIS

### DISCUSSION THREAD

**Discussion by fbelli997** [NatLabRockies/floris#988] [General]
Certo! Ecco il testo corretto:

Hi everybody,

I'm writing on the forum to ask a question about the superposition of wakes.
I was trying to understand how FLORIS computes the deficit and wake superposition, and I realized that after the calculation of the deficit, in the solver.py script, the wake field is computed by multiplying the deficit with the freestream velocity for each turbine.
This seems odd to me because I was expecting the wake field to be calculated using the wind speed experienced by each turbine.
Is there a specific reason why the wake is calculated this way?
Below is the part of the code I am referring to, at line 217 of the script:

`        wake_field = model_manager.combination_model.function(
            wake_field,
            velocity_deficit * flow_field.u_initial_sorted
        )`

Thank you all in advance!

**Comment by paulf81** [comment]
Hi @fbelli997 , I believe this question is also discussed in #948 , would you check that response and see if it answers your question?  Thanks!
